README.md

```diff
@@ -1,70 +1,196 @@
+![React Router](https://img.shields.io/badge/React_Router-CA4245?style=for-the-badge&logo=react-router&logoColor=white)
+![Cypress](https://img.shields.io/badge/Cypress-17202C?logo=cypress&logoColor=fff&style=for-the-badge)
+![GraphQL](https://img.shields.io/badge/GraphQl-E10098?style=for-the-badge&logo=graphql&logoColor=white)
 
-In the project directory, you can run:
 
-### `npm start`
+# 🧠 Contributors
+Erica Hagle 
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+[![LinkedIn][linkedin-shield]][linkedin-url-eh] 
+[![GitHub][github-shield-eh]][github-url-eh]
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
 
-### `npm test`
+Arden Ranta 
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+[![LinkedIn][linkedin-shield]][linkedin-url-ar] 
+[![GitHub][github-shield-ar]][github-url-ar]
 
-### `npm run build`
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+Xander Hendry 
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+[![LinkedIn][linkedin-shield]][linkedin-url-zh]
+[![GitHub][github-shield-zh]][github-url-zh]
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
 
-### `npm run eject`
+Sam Tran 
 
-**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+[![LinkedIn][linkedin-shield]][linkedin-url-st]
+[![GitHub][github-shield-st]][github-url-st]
 
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
+Kevin Zolman 
 
-You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+[![LinkedIn][linkedin-shield]][linkedin-url-kz]
+[![GitHub][github-shield-kz]][github-url-kz]
 
-## Learn More
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+# 💭 Abstract
+<!-- TBD -->
 
-To learn React, check out the [React documentation](https://reactjs.org/).
+# 📝 Context
```

Comment:
I really like the context section and will nab this for the backend readme. I will be sure to link the frontend repo as you did for the backend.